specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/examples/connectionsUpdate.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testResourceGroup",
+    "connectionName": "testManagedApi",
+    "api-version": "2016-06-01",
+    "apiConnection": {
+      "tags": {
+        "sampleTag": "SampleValue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "displayName": "testManagedApi",
+          "statuses": [
+            {
+              "status": "Connected"
+            }
+          ],
+          "customParameterValues": {},
+          "nonSecretParameterValues": {
+            "integrationAccountId": "/subscriptions/f34b22a3-2202-4fb1-b040-1332bd928c84/resourceGroups/testResourceGroup/providers/Microsoft.Logic/integrationAccounts/testIntegrationAccount"
```

Comment:
I am not sure what is the meaning of these examples.

But the nonSecretParameterValues and customParameterValues are mainly objects and they can any properties. So modify them accordignly. Does this make integrationAccountId as a required parameter?

Reply:
That's an interesting point. That's in the response body, so it wouldn't be required, it just happened to be on the connection I grabbed as an example. They are just dictionaries of strings, so they hypothetically could be anything. Do you have a preference for how you would like it changed?

Reply:
I suppose you can just leave it empty. But these could essentially be any collection of key-value pairs